lib/events.js

```diff
@@ -235,22 +235,20 @@ function _addListener(target, type, listener, prepend) {
     }
 
     // Check for listener leak
-    if (!existing.warned) {
-      m = $getMaxListeners(target);
-      if (m && m > 0 && existing.length > m) {
-        existing.warned = true;
-        // No error code for this since it is a Warning
-        // eslint-disable-next-line no-restricted-syntax
-        const w = new Error('Possible EventEmitter memory leak detected. ' +
-                            `${existing.length} ${String(type)} listeners ` +
-                            'added. Use emitter.setMaxListeners() to ' +
-                            'increase limit');
-        w.name = 'MaxListenersExceededWarning';
-        w.emitter = target;
-        w.type = type;
-        w.count = existing.length;
-        process.emitWarning(w);
-      }
+    m = $getMaxListeners(target);
+    if (m && m > 0 && existing.length > m && !existing.warned) {
```

Comment:
If you have time and inclination to test: does reducing `m && m > 0` to `m > 0` make a difference?